unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/InternalECSComponents.cs

```diff
@@ -20,6 +23,18 @@ public InternalECSComponents(ECSComponentsManager componentsManager, ECSComponen
             componentsManager,
             componentsFactory,
             () => new InternalMaterialHandler());
+
+        onPointerColliderComponent = new InternalECSComponent<InternalColliders>(
+            InternalECSComponentsId.COLLIDER_POINTER,
+            componentsManager,
+            componentsFactory,
+            null);
```

Comment:
these won't have a handler?

Reply:
since we don't need to react to component update/create/remove, we are using only it's data, we don't need a handler for them